ui/src/i18n/de.json

```diff
@@ -103,11 +103,11 @@
     "login.title_login": "Anmeldung",
     "logs.label_download": "Download",
     "logs.label_truncate": "Truncate",
-    "logs.message_download_warning": "WARNING: Homebridge log files may contain personal information, passwords or access tokens. Please review the log file and remove any sensitive information before sharing with others.",
-    "logs.message_log_file_truncated": "Log file truncated.",
-    "logs.message_truncate_log_warning": "This action will irreversibly delete your existing Homebridge logs. Are you sure you want to continue?",
-    "logs.title_download_log_file": "Download Log File",
-    "logs.title_truncate_log_file": "Truncate Log File",
+    "logs.message_download_warning": "ACHTUNG: Homebridge-Logdateien können persönliche Informationen, Passwörter or Zugrifftokens beinhalten. Bitte sieh dir die Logdatei an und entferne alle sensitiven Informationen, bevor du sie mit anderen teilst.",
```

Comment:
`or` should be translated as well to `oder` maybe `und/oder` would fit even better.